src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml

```diff
@@ -7,8 +7,11 @@
     </button>
     <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bulk-action-menu-button">
         @{
-            dynamic bulkActions = await ContentOptionsDisplayManager.BuildDisplayAsync(Model, null, "BulkActions");
-            @await DisplayAsync(bulkActions.Content)
+            var bulkActions = await ContentOptionsDisplayManager.BuildDisplayAsync(Model, null, "BulkActions");
+            if (bulkActions.TryGetProperty<IShape>("Content", out var content))
```

Comment:
Do we have to call "Try" here?

Reply:
We can call `Property<IShape>()` but what if it does not exists?

Reply:
OK, let's assume the Content zone may not be created after BuilddisplayAsync.